src/hooks/reactQuery/useRag.ts

```diff
@@ -56,6 +57,44 @@ export const useChat = () => {
   });
 };
 
+export const useGetAgentChatHistory = (agentId: string | undefined) => {
+  const wallet = useWallet();
+
+  return useQuery({
+    queryKey: [`${QUERY_KEYS.AGENT_CHATS}-${agentId}`],
```

Comment:
is there a reason we can doing it like it instead of 
`queryKey: [QUERY_KEYS.AGENT_CHATS,agentID]`